tests/functional/tests.py

```diff
@@ -1799,215 +1922,222 @@ def main():
             test_list_objects_v2(client, log_output)
 
-            log_output =  LogOutput(client.presigned_get_object, 'test_presigned_get_object_default_expiry')
+            log_output = LogOutput(client.presigned_get_object, 'test_presigned_get_object_default_expiry')
             test_presigned_get_object_default_expiry(client, log_output)
 
-            log_output =  LogOutput(client.presigned_get_object, 'test_presigned_get_object_expiry_5sec')
+            log_output = LogOutput(client.presigned_get_object, 'test_presigned_get_object_expiry_5sec')
             test_presigned_get_object_expiry_5sec(client, log_output)
 
-            log_output =  LogOutput(client.presigned_get_object, 'test_presigned_get_object_response_headers')
+            log_output = LogOutput(client.presigned_get_object, 'test_presigned_get_object_response_headers')
             test_presigned_get_object_response_headers(client, log_output)
 
-            log_output =  LogOutput(client.presigned_put_object, 'test_presigned_put_object_default_expiry')
+            log_output = LogOutput(client.presigned_put_object, 'test_presigned_put_object_default_expiry')
             test_presigned_put_object_default_expiry(client, log_output)
 
-            log_output =  LogOutput(client.presigned_put_object, 'test_presigned_put_object_expiry_5sec')
+            log_output = LogOutput(client.presigned_put_object, 'test_presigned_put_object_expiry_5sec')
             test_presigned_put_object_expiry_5sec(client, log_output)
 
-            log_output =  LogOutput(client.presigned_post_policy, 'test_presigned_post_policy')
+            log_output = LogOutput(client.presigned_post_policy, 'test_presigned_post_policy')
             test_presigned_post_policy(client, log_output)
 
-            log_output =  LogOutput(client.put_object, 'test_thread_safe')
+            log_output = LogOutput(client.put_object, 'test_thread_safe')
             test_thread_safe(client, testfile, log_output)
 
-            log_output =  LogOutput(client.get_bucket_policy, 'test_get_bucket_policy')
+            log_output = LogOutput(client.get_bucket_policy, 'test_get_bucket_policy')
             test_get_bucket_policy(client, log_output)
 
-            log_output =  LogOutput(client.set_bucket_policy, 'test_set_bucket_policy_readonly')
+            log_output = LogOutput(client.set_bucket_policy, 'test_set_bucket_policy_readonly')
             test_set_bucket_policy_readonly(client, log_output)
 
-            log_output =  LogOutput(client.set_bucket_policy, 'test_set_bucket_policy_readwrite')
+            log_output = LogOutput(client.set_bucket_policy, 'test_set_bucket_policy_readwrite')
             test_set_bucket_policy_readwrite(client, log_output)
 
             log_output = LogOutput(client.get_bucket_notification, 'test_get_bucket_notification')
             test_get_bucket_notification(client, log_output)
 
+            if sys.version_info.major == 3:
```

Comment:
Can we add a comment on why we running on 3+ only ?

Reply:
done.